test/check-machines-consoles

```diff
@@ -254,5 +258,76 @@ fullscreen=0
                                   "Failed when connecting: Connection closed")
 
 
+class TestMultiMachinesConsoles(ConsoleCase):
+    provision = {  # noqa: RUF012
+        "machine": {"address": "10.111.113.1/20"},
+    }
```

Comment:
This is still only one machine -- seems you could get this more cheaply with the already running nondestructive machine? `172.27.0.15` is reliable, a couple of cockpit tests refer to it as well.

Reply:
Right, I thought I'd seen something like that... Will that work also with TF?

Reply:
I have made the new test destructive and use that address.